.gitignore

```diff
@@ -1 +1,3 @@
-*.pyc
\ No newline at end of file
+*.pyc
+*college/
+*settings.py
```

Comment:
Seriously settings.py. You can't add that. And what is this college/ ? If you have these things locally, have these changes locally in the .gitignore. Don't push them.